### ISSUE TITLE
Release 1.4

### ISSUE THREAD

**Issue by phamtrinli** [adrianhurt/play-bootstrap#111]
Hi,

thanks for this project :-)
Any plans when version 1.4 is released on Sonatype? So far I've just found snapshot versions.

Kind regards,
Christian

**Comment by nob13** [comment]
forked the repo and released 1.4-P26-B4, hope this is only needed temporary ;)

**Comment by nafg** [comment]
Pretty please? ;)

**Comment by mkurz** [comment]
@adrianhurt [Play 2.7.0-RC3 released](https://discuss.lightbend.com/t/play-2-7-0-rc3-released/2715) - would nice if we see a new release of play-bootstrap

**Comment by mkurz** [comment]
@adrianhurt [Play 2.7.0 released](https://blog.playframework.com/play-2-7-0-is-here/).
Will you continue maintaining play-bootstrap?

**Comment by adrianhurt** [comment]
Hi, sorry for that. It’s my intention but it’s true I don’t get enough time these last months. I’ll try to give it a bump within the next 2 weeks.

**Comment by mkurz** [comment]
Thanks @adrianhurt! I really appreciate the time you invest into this library!

**Comment by mkurz** [comment]
@adrianhurt Please also add support for Play 2.7 and please also provide builds for Scala 2.13.0-M5. I opened a ticket for this, see #115. Thanks!

**Comment by adrianhurt** [comment]
Hi! Again, sorry for being away for so much time! But finally, I've just released version 1.4. I've simply updated some libraries, so I hope everything is fine. But anyway just tell me if you see something wrong and I promise to correct it quickly.

So the released versions are:
* 1.4-P6-B4
* 1.4-P6-B3
* 1.4-P5-B4
* 1.4-P5-B3
* 1.5-P27-B4-SNAPSHOT
* 1.5-P26-B4-SNAPSHOT

I'll try to update the docs the next week.
